xl/metadata/asf.py

```diff
@@ -33,14 +33,14 @@ class AsfFormat(BaseFormat):
     MutagenType = asf.ASF
     # TODO: figure out the the WM/ prefix is universal
     tag_mapping = {
-        "artist"        : "Author",
-        "album"         : "WM/AlbumTitle",
-        "title"         : "Title",
-        "genre"         : "WM/Genre",
-        "tracknumber"   : "WM/TrackNumber",
-        "date"          : "WM/Year",
-        "albumartist"   : "WM/AlbumArtist",
-        "grouping"      : "WM/ContentGroupDescription"
+        "artist": "Author",
```

Comment:
This breaks the indentation. I won't mind but as far as I remember one of you (@sjohannes or @virtuald) did.

Reply:
This repeats multiple times.

Reply:
Good catch, would have to add fmt off directives around this.

Reply:
There are a few examples of this all over the code base.